searx/data/engine_descriptions.json

```diff
@@ -992,6 +1058,10 @@
   "brave":[
    "The Brave browser is a fast, private and secure web browser for PC, Mac and mobile. Download now to enjoy a faster ad-free browsing experience that saves data and battery life by blocking tracking software.",
    "https://brave.com/search/"
+  ],
+  "onesearch":[
+   "OneSearch is a free privacy-oriented search experience that lets you explore what you care about without unwanted profile or ad targeting. Take back control of your data today!",
+   "https://www.onesearch.com"
```

Comment:
The script `./searxng_extra/update/update_engine_descriptions.py` has found a documentation / even there exists no wikidata entry.